fili-core/src/main/java/com/yahoo/bard/webservice/data/HttpResponseMaker.java

```diff
@@ -136,7 +136,7 @@ private ResponseBuilder createResponseBuilder(
                                 LinkedHashMap::new
                         ));
 
-        Response response = new Response(
+        ResponseData response = new ResponseData(
```

Comment:
You're building a `ResponseData` object, but you're not actually using a `ResponseWriter` anywhere (except through the `ResponseData::write` method that should be removed). In order to be effective, the `ResponseWriter` needs to be injected, just like the `HttpResponseMaker`:

1. Add a binding of `ResponseWriter` to the `AbstractBinderFactory` that defaults to the `FiliResponseWriter`.
2. Pass the `ResponseWriter` into the `HttpResponseMaker`. The injection framework should then get the writer into the maker.
3. Use the writer to write the response to the outputstream.
4. Pass the output stream to the [response builder](https://github.com/yahoo/fili/pull/455/files#diff-7bdeb7a580261f6a4a65dd6bf01df1abR154)